LambdaEngine/Source/Application/API/CommonApplication.cpp

```diff
@@ -28,12 +29,12 @@ namespace LambdaEngine
 		// Create platform application
 		m_pPlatformApplication = pApplication;
 		m_pPlatformApplication->SetEventHandler(s_CommonApplication);
-
+		
```

Comment:
Trailing whitespace.

I found an extension for visual studio that removes whitespaces on save:
https://marketplace.visualstudio.com/items?itemName=MadsKristensen.TrailingWhitespaceVisualizer